src/bioregistry/curation/curated_papers.csv

```diff
@@ -1 +1,21 @@
-pmid, relevant, relevancy_type, notes
\ No newline at end of file
+pmid,relevant,relevancy_type,notes
+39104285,1,new_provider,"Provider for UniProt IDs, issue with curation due to multiple URI formats depending on plant species"
+39074139,1,new_provider,"Resolver for PDB IDs"
+39014503,0,no_website,
+39047988,0,irrelevant_other,
+39115390,0,irrelevant_other,
+39095357,0,irrelevant_other,
+39084442,0,not_identifiers_resource,
+38991851,1,unclear,"identifiers for sharing, retrieving, and validating sample metadata. Unclear if this should be curated as a prefix, provider or a separate registry"
+38991828,0,irrelevant_other,
+39049520,0,not_identifiers_resource,
+39104826,1,existing,"Already present in the bioregistry as a provider for mesh prefix"
+39050757,0,irrelevant_other,
+39064021,0,irrelevant_other,
+39028894,0,not_identifiers_resource,
+39044201,0,not_identifiers_resource,"Potential resource for rare diseases identifiers, but not identifier information"
+39088253,0,irrelevant_other,
+39119155,0,irrelevant_other,
+39005357,0,irrelevant_other,
+39044130,0,irrelevant_other,
+39010878,0,irrelevant_other,
```

Comment:
can you add a column with your ORCID and the date when you did curations?

is there some reference material about the `relevancy_type`? If not, this PR would be a good place to write a markdown file with a full explanation of what this file is, what all of the columns arem how it was generated, and what's in it

Reply:
| **relevancy_type** | **definition** |
|--------------|--------------|
| new_prefix        | A resource for new primary identifiers       |
| new_provider      | A resolver for existing identifiers        |
| new_publication      | A new publication for an existing prefix       |
| not_identifiers_resource      | A database, but not for identifier information      |
| no_website       | Paper suggestive of a new database, but no link to website provided      |
| existing       | An existing entry in the bioregistry        |
| unclear      |  Not clear how to curate in the bioregistry, follow up discussion required     |
| irrelevant_other       | Completely unrelated information       |

This is the working list of terms I am using to label papers. I would appreciate any feedback or corrections on how I can make it more descriptive or concise to ensure that anyone other than myself can use this is a reference in the future.

Reply:
I agree, we can extend the CSV file with an orcid column and a date. The tags are still evolving as we start looking at papers but I like these initial tags. As @cthoyt suggested, pasting the table from your comment above into a README.md file + a bit of text explaining which CSV file it is relevant for, then adding it to version control would be useful.

Reply:
Then next step is also to add unit tests to ensure the integrity of this file, e.g.

1. checking that required fields are present
2. checking format of data is right (PMID is an integer, ORCID matches specific regex pattern, dates are in YYYY-MM-DD standard format)
3. checking enumeration for relevancy_type is part of a controlled vocabulary